libpod/container_api.go

```diff
@@ -201,6 +196,7 @@ func (c *Container) Kill(signal uint) error {
 		}
 	}
 
+	// TODO: Is killing a paused container OK?
```

Comment:
A paused container shouldn't be running, so I'm not sure how you'd stop the process there.  If you could, I don't think you should unless there was some kind of `--force` option added to the kill command.

Reply:
Paused processes can still receive signals, which has some interesting applications - you can pause a container, SIGKILL every process in it, and unpause it, to cause it to immediately exit without any possibility of a race condition around `fork()`

Reply:
Thanks for the 411, I wasn't aware that paused containers could receive signals still.